src/flutter/shell/platform/linux_embedded/window/native_window_drm_eglstream.cc

```diff
@@ -0,0 +1,299 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux_embedded/window/native_window_drm_eglstream.h"
+
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#include "flutter/shell/platform/linux_embedded/logger.h"
+#include "flutter/shell/platform/linux_embedded/surface/cursor_data.h"
+
+namespace flutter {
+
+NativeWindowDrmEglstream::NativeWindowDrmEglstream(const char* deviceFilename) {
```

Comment:
You can make the constructor common by renaming from ConfigureDisplayForEglstream to ConfigureDisplayAdditional. In the drm-gbm case, ConfigureDisplayAdditional does nothing.

Reply:
I have added a constructor to `NativeWindowDrm` that summarizes the common processing of GBM and EGLStream. Each specific process is executed in each constructor.